baseline/client/tests/task-classes.test.js

```diff
@@ -38,7 +38,15 @@ describe.each([
     
     it(`${name} has taskName`, () => {
         expect(typeof taskClass.taskName).toBe("string");
-        const inst = new taskClass(1);
+        const args = (() => {
```

Comment:
There's a merge conflict here that looks like your recent changes for verbal fluency weren't included. Can you rebase the branch for this and update the PR?